docker-compose.yml

```diff
@@ -479,6 +497,7 @@ services:
 
   kopano_scheduler:
     image: ${docker_repo:-zokradonh}/kopano_scheduler:${SCHEDULER_VERSION:-latest}
+    restart: unless-stopped
```

Comment:
As already commented I would rather go with `restart: no` here

Reply:
Ok i know the hassle with endlessly restarting services, when there is some problem on startup. Specifying the `restart: no` will result in a not-running scheduler, when the docker host gets an reboot. So when the scheduler is used to create backups, this should be treated carefully, since this will disable backups as soon as the admin restarts the host.

When i was debugging or fixing not properly starting services, i used to stop them manually. So calling manually `docker-compose stop kopano_scheduler`, will prevent it from automatically restarting. So this usage mimics my real wanted solution of an restart unless stopped or failed, which is sadly not available on the docker daemon.

@fbartels do you think that the risk of host restart disabled backups are treated with enough care, when adding some documentation?

Reply:
I'm sorry, it seems the "no" needs to be quoted.

From experience I know: people don't read documentation ;-) so the only thing that protects you from containers not running, and backups not happening is monitoring, monitoring, monitoring.